pkg/coderefs/coderefs.go

```diff
@@ -60,6 +62,10 @@ type branch struct {
 }
 
 func Scan() {
+	log.Stdout.Printf("ld-find-code-refs version " + version)
```

Comment:
its annoying but we should probably move this up higher in the callstack. may require duplicating it to each entrypoint. but, consider:

`cmd/ld-find-code-refs/main.go`:
```
func main() {
	err, cb := o.Init()
	if err != nil {
		log.Init(false)
		log.Error.Printf("could not validate command line options: %s", err)
		cb()
		os.Exit(1)
	}
	log.Init(o.Debug.Value())
	coderefs.Scan()
}
```
Someone can't figure out how to get their params set up right, so they submit a support ticket with the output. We won't automatically see their version number.

Drawback of this is we'll need to duplicate this log statement in each entrypoint (GA, bitbucket, cmd, etc)

Reply:
counterargument is its better to just keep it in one place, and we can always ask someone to just re-run it with `-version` and submit that

Reply:
You bring up a good point. I also just realized that `--version` won't work if it's the only parameter sent over, because the program fails on the required arguments, which weren't provided.

This PR needs a little work to make --version usable